python/semantic_kernel/agents/bedrock/models/bedrock_agent_event_type.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+from enum import StrEnum
```

Comment:
this is not supported in Python 3.10, see https://docs.python.org/3/library/enum.html#enum.StrEnum